### ISSUE TITLE
iOS: Stuck at Black Screen 

### ISSUE THREAD

**Issue by mbakgun** [koush/vysor.io#702]
**Describe the bug**
I use Vysor for mac, and it just shows Black Screen after "Waiting for Decoder..." text, but I can write text, and also press the home button from this screen nicely. Can screenshot etc. But the view doesn't appear. 

**To Reproduce**
1) Connecting iPhone 5s to Mac via USB
2) Click view on available devices
3) Wait and just see the "Waiting for Decoder..." Screen
4) Then black hole taking me inside 

**Expected behavior**
I used to Vysor for more than 3 years on Android devices and started to use it in ios yesterday. I expect to work on iOS, like Android devices. 

**Screenshots**
<img width="543" alt="Screen Shot 2021-01-22 at 09 32 18" src="https://user-images.githubusercontent.com/8726393/105455572-c5e58280-5c94-11eb-8c55-6a0972e22dc0.png">

**Mac (please complete the following information):**
 - MacBook Pro (16-inch, 2019)
 - OS: MacOS Catalina - 10.15.7
 - Vysor Version 4.0.27

**iOS:**
 - Device: iPhone 5S
 - OS 12.5.1
 - Vysor Bug Report Link: https://gist.github.com/bef13b61fbc1db50f37855f147c52ccd

**Comment by koush** [comment]
Just to confirm, the iPhone screen is on and on the home screen?
The iPhone 5s is very old, and truth be told, I have no idea if it works. It may not.

**Comment by koush** [comment]
I have an iPhone 4… I’ll see if that works. Currently I have confirmation that it works as far back as iPhone 8.

**Comment by mbakgun** [comment]
Yes, I agree it's an ancient device in 2021. However, our application provides support, and we need to test at 5S :)

**Comment by koush** [comment]
> Yes, I agree it's an ancient device in 2021. However, our application provides support, and we need to test at 5S :)

Any way you can confirm it is an iPhone 5 issue? Do you have more recent iPhones on hand?

**Comment by mbakgun** [comment]
Due to Covid 19 restrictions, unfortunately, I have no chance to check it soon.
There are alternatives you might know. Which using airplay services to share & control content. I not really deep dived as an Android developer. But I saw there are also paid solutions. Which goes well with 5S.

 Any chance to implement it for future developments? I would love to test it.

**Comment by koush** [comment]
The wired version is using airplay over the wire essentially. I think there's some older device specific quirks I just need to watch out for.

**Comment by mbakgun** [comment]
Thanks looking forward to future software updates.

**Comment by miguelnunorosa** [comment]
Hi.

I've osx 10.15.7 and when I try with my Android device (Xiaomi Redmi 8) I stuck on blackscreen with text "Waiting for Decoder". How I resolve this?

Thank you.

**Comment by koush** [comment]
Confirmed on iPhone 4. unclear what the issue is.

**Comment by mbakgun** [comment]
Thanks, @koush,
I've recently recognized that rotation (landscape & portrait) works too.  

Well, will you close this issue or try to fix it? Is there any hope for dated devices?

**Comment by koush** [comment]
Will try to fix it. It seems to be a decoder issue. Will see.

**Comment by MisanEnemigin** [comment]
I have a similar error.
Mine says 'Waiting For Decoder'

Model:
Mac Os X Yosemite
10.10.5

Device:
Nokia 2.4

**Comment by mbakgun** [comment]
Hi @koush  , 
Do you have any updates?  Btw I can confirm, app works with IPhone 6.

**Comment by jonsaw** [comment]
I'm using an iPhone Xr, I'm also getting a black screen.

**Comment by gibahjoe** [comment]
I have an iPhone XR too. @jonsaw How do I get the beta channel? I downloaded the app from the official website

**Comment by koush** [comment]
Fixed in stable channel too now.

**Comment by AugustoAleGon** [comment]
![image](https://user-images.githubusercontent.com/14319083/129620681-d3a86b47-63b2-4f4a-bca9-01e89bed3b85.png)
I am getting a black screen when I tried to use it on iPhone 8S with iOS 14.7.1

**Comment by AugustoAleGon** [comment]
@koush is there any updates on this?

**Comment by ninora** [comment]
Having the same problem with iPhone X ver 14.7.1

**Comment by AugustoAleGon** [comment]
@ninora What works for me. What turn off and on the bluetooth. Try that and let me know.

**Comment by ninora** [comment]
@AugustoAleGon Same thing.

Here's what I do after I installed Windows drivers:

1. iPhone (model X ver 14.7.1) showing home page, Bluetooth is on
2. Start Vysor app on the phone and click Start Broadcast
3. Start Vysor on Windows 10 (64 bit)
4. Click the red start button for iOS Device (showing the iphone ID)
5. Screen opens and says "Waiting for Bluetooth device..."

Things I tried:
- Turning BT on off and on
- With and w/out Vysor app on the phone
- Disconnecting and reconnecting phone from PC
